doc/BENCHMARKING.md

```diff
@@ -39,10 +39,17 @@ You can use the following steps.
         export S3_BUCKET_BENCH_FILE=bench_file_name
         export S3_BUCKET_SMALL_BENCH_FILE=small_bench_file_name
 
-3. Create the bench files manually in your bucket. The size of the files must be exactly the same as the size defined in fio configuration files. The easiest way to do this is running fio against your local file system first to let fio create the files for you, and then upload them to your S3 bucket using the AWS CLI. For example:
-
-        fio --directory=your_local_dir --filename=your_file_name mountpoint-s3/scripts/fio/read/seq_read_small.fio
-        aws s3 cp your_local_dir/your_file_name s3://${S3_BUCKET_NAME}/${S3_BUCKET_TEST_PREFIX}
+3. Create the bench files manually in your bucket(create a `benchmark` directory in it if not present). The size of the files must be exactly the same as the size defined in fio configuration files. The easiest way to do this is mounting your bucket on local file system using mountpoint-s3. Then, running fio against your mount directory to let fio create the files for you.
+
+        mount-s3 your-bucket-name your_mount_dir/ --allow-delete --prefix benchmark/
```

Comment:
We don't need `--allow-delete`. Also, we use `DOC-EXAMPLE-BUCKET` and `/path/to/mount` in other documents. Probably use the same for consistency.
```suggestion
        mount-s3 DOC-EXAMPLE-BUCKET /path/to/mount --prefix benchmark/
```

Reply:
Oh, I forget one thing that we will need `--part-size` because the default part size won't be enough to create a 100GB file.